tasks/src/audio-classification/data.ts

```diff
@@ -58,6 +58,7 @@ const taskData: TaskDataCustom = {
 			id:          "superb/hubert-large-superb-er",
 		},
 	],
+	spaces:  [],
```

Comment:
What are the keys for the dict in this list? description & id?

Reply:
it's an `ExampleRepo` same as the other repo types